integration_test/kafka_batching/kafka_batching_test.go

```diff
@@ -0,0 +1,478 @@
+package kafka_batching
```

Comment:
**Question:** does batching work in combination with **AVRO Schema Lists** and/or **multiple topic support**? This is not evident by looking [here](https://github.com/rudderlabs/rudder-transformer/blob/develop/src/v0/destinations/kafka/transform.js), where it seems that the `process` method performs more steps than its `batch` counterpart. Maybe `batch` is called with events that have already been `process`ed in a previous step?

@utsabc @koladilip

Reply:
I think I can answer that question myself... events are first transformed in processor, then batched in router!